packages/editor/src/pages/Editor/store/solutions/sagas.ts

```diff
@@ -11,8 +11,11 @@ import { SCRIPT_FILE_NAME, NULL_SOLUTION_ID, NULL_FILE_ID } from '../../../../co
 import { deleteSolutionFromStorage } from '../localStorage';
 import { formatTypeScriptFile } from '../editor/utilities';
 import { currentOfficeJsRawSnippetsBaseRepoUrl } from 'common/lib/environment';
+import { actions as host } from 'script-lab-core/lib/modules/host';
 
 export default function* solutionsWatcher() {
+  yield takeEvery(getType(host.change), openLastOpenedOrBackstageSaga);
```

Comment:
This needed to be added due to the fact that the functionality used to exist in the local host sagas, but doesn't in script-lab-core's host sagas.